style.css

```diff
@@ -22,7 +22,21 @@
 }
 
 /* q */
-.dokuwiki q.blockquote-plugin {
+.dokuwiki q.blockquote-plugin {\0022
```

Comment:
this gives a default behavior of double quotes for first level quotes tag, then second level (embeded) renders as single quotes but you could change this behavior using any of the entity codes given in the comment. This standard is documented [here](https://www.w3schools.com/CSSref/pr_gen_quotes.asp)